src/coreclr/vm/commodule.cpp

```diff
@@ -862,15 +862,15 @@ void QCALLTYPE COMModule::GetFullyQualifiedName(QCall::ModuleHandle pModule, QCa
         if (*fileName != 0) {
                 retString.Set(fileName);
         } else {
-            hr = UtilLoadStringRC(IDS_EE_NAME_UNKNOWN, wszBuffer, sizeof( wszBuffer ) / sizeof( WCHAR ), true );
+            hr = UtilLoadStringRC(W("<Unknown>"), wszBuffer, sizeof( wszBuffer ) / sizeof( WCHAR ), true );
```

Comment:
This should be just `retString.Set(W("<Unknown>"))`. It should not be calling `UtilLoadStringRC` at all.

Reply:
Also, delete `wszBuffer` local in this method.

Reply:
duh, thanks